src/components/Menu/index.test.tsx

```diff
@@ -1,48 +1,62 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 
 import { MenuItem } from 'components';
 import Menu from '.';
 
-describe('Popover component', () => {
+describe('Menu component', () => {
   /**
    * `@reach/menu-button` has its own test coverage.
    * FDS provides only styling overrides, so snapshots will be adequate coverage for our code.
    */
-  it('matches snapshot (string items)', () => {
-    const wrapper = mount(
+  it('renders component (strings)', () => {
+    render(
       <Menu trigger={<a href="#">trigger</a>}>
-        <MenuItem onSelect={() => {}}>First</MenuItem>
-        <MenuItem onSelect={() => {}}>Second</MenuItem>
-        <MenuItem onSelect={() => {}}>Third</MenuItem>
+        <MenuItem onSelect={jest.fn()}>First</MenuItem>
+        <MenuItem onSelect={jest.fn()}>Second</MenuItem>
+        <MenuItem onSelect={jest.fn()}>Third</MenuItem>
       </Menu>
     );
-    expect(wrapper).toMatchSnapshot();
+    expect(screen.getByText('First')).toBeInTheDocument();
+    expect(screen.getByText('Second')).toBeInTheDocument();
+    expect(screen.getByText('Third')).toBeInTheDocument();
   });
-  it('matches snapshot (JSX items)', () => {
-    const wrapper = mount(
+  it('renders component (JSX items)', () => {
+    render(
       <Menu trigger={<a href="#">trigger</a>}>
-        <MenuItem onSelect={() => {}}>
+        <MenuItem onSelect={jest.fn()}>
           <p>First</p>
         </MenuItem>
-        <MenuItem onSelect={() => {}}>
+        <MenuItem onSelect={jest.fn()}>
           <p>Second</p>
         </MenuItem>
-        <MenuItem onSelect={() => {}}>
+        <MenuItem onSelect={jest.fn()}>
           <p>Third</p>
         </MenuItem>
       </Menu>
     );
-    expect(wrapper).toMatchSnapshot();
+    expect(screen.getByText('First')).toBeInTheDocument();
+    expect(screen.getByText('Second')).toBeInTheDocument();
+    expect(screen.getByText('Third')).toBeInTheDocument();
   });
-  it('matches snapshot (disabled)', () => {
-    const wrapper = mount(
-      <Menu isDisabled trigger={<a href="#">trigger</a>}>
-        <MenuItem onSelect={() => {}}>First</MenuItem>
-        <MenuItem onSelect={() => {}}>Second</MenuItem>
-        <MenuItem onSelect={() => {}}>Third</MenuItem>
+  it('renders trigger element only', () => {
+    const mockOnClick = jest.fn();
+    render(
+      <Menu
+        isDisabled
+        trigger={
+          <a href="#" onClick={mockOnClick}>
+            trigger
+          </a>
+        }
+      >
+        <MenuItem onSelect={jest.fn()}>First</MenuItem>
+        <MenuItem onSelect={jest.fn()}>Second</MenuItem>
+        <MenuItem onSelect={jest.fn()}>Third</MenuItem>
       </Menu>
     );
-    expect(wrapper).toMatchSnapshot();
+    const anchor = screen.getByRole('link');
+    expect(anchor).toBeInTheDocument();
+    expect(screen.queryAllByText(/first|second|third/i).length).toStrictEqual(0);
```

Comment:
Is there another way to write this last test? The way it's written, I don't really get what this is doing, and why.

Reply:
what is doing is, looking for the `MenuItem`s and asserting they're not there, because when `Menu` is disabled it just returns the `trigger`, of the top of my head I could query for each of them individually and check for them to be null.

if you have a different idea in mind, i'm all eyes.